packages/entities/entities-shared/src/components/entity-base-form/EntityBaseForm.vue

```diff
@@ -42,6 +42,49 @@
         data-testid="form-actions"
       >
         <slot name="form-actions">
+          <KToggle
+            v-if="config.jsonYamlMilestone2Enabled"
+            v-slot="{ isToggled, toggle }"
+          >
+            <div>
+              <KButton
+                appearance="tertiary"
+                data-testid="form-view-configuration"
+                @click="toggle"
+              >
+                {{ t('baseForm.actions.viewConfiguration') }}
+              </KButton>
+              <KSlideout
```

Comment:
https://github.com/Kong/public-ui-components/assets/6960750/bfdc1dfb-8cb4-4309-b084-35b697ee9dfb

This KSlideout component should be moved outside the `form-actions` area. Otherwise, canceling (closing) the form triggers a form submission

Reply:
I used `@click.prevent` on the `View Configuration` button to prevent form submission.

Unable to move the KSlideout component outside as it wraps the `View Configuration` button which we want to align with the `Submit` and `Cancel` form action buttons, and even tried using `Teleport` but it required the KSlideout to be rendered before being mounted.

Reply:
Moved the KSlideout component outside the `form-actions` area